src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -162,9 +162,11 @@ public String getScript() {
 
         private int pid;
         private final long startTime = System.currentTimeMillis();
+        private EnvVars envVars;
```

Comment:
This is going to be null after deserialization of a `sh` step in a build that started running prior to plugin upgrade, so you must handle that.